### PR TITLE
Add splash pages for Europe (en-EU) and Ireland (en-IE)

### PR DESCRIPTION
**Pull request by Ken-Lo**
Adding Republic of Ireland (https://gocardless.com/en-ie/):
screen shot 2015-05-22 at 17 59 50

and Europe (https://gocardless.com/en-eu/):
screen shot 2015-05-22 at 18 00 12

### CODE REVIEW

File: app/pages/faq/merchants/customer-experience/faq-merchants-customer-experience.en.js [hunk 1]
```diff
@@ -8,27 +8,37 @@ export default class FaqMerchantsCustomerExperienceEn extends React.Component {
     return (
       <Translation locales='en'>
```
Comment (harrison) [inline app/pages/faq/merchants/customer-experience/faq-merchants-customer-experience.en.js]: should the route be restricted to `en-GB`? there's now only a header for `en-IE`

File: app/pages/faq/merchants/developer-api/faq-merchants-developer-api.en.js [hunk 2]
```diff
@@ -13,8 +13,13 @@ export default class FaqMerchantsDeveloperApiEn extends React.Component {
...
         </h3>
         <p className='para'>
-          Our REST API allows developers to easily create powerful integrations with GoCardless. See
-          our <a href='https://developer.gocardless.com/' className='u-link-color-p u-text-underline'>documentation</a> to find out more.
+          Our REST API allows developers to easily create powerful integrations with GoCardless.
+          <Translation locales='en-GB'>
+            See our <a href='https://developer.gocardless.com/' className='u-link-color-p u-text-underline'>documentation</a> to find out more.
+          </Translation>
+          <Translation locales={['en']} exclude={['en-GB']}>
+            See our <a href='https://developer.gocardless.com/pro' className='u-link-color-p u-text-underline'>documentation</a> to find out more.
```
Comment (harrison) [inline app/pages/faq/merchants/developer-api/faq-merchants-developer-api.en.js]: could be in messages instead as `documentation_link` and use the `Href` component to create the link

File: app/pages/faq/merchants/direct-debit/faq-merchants-direct-debit.en.js [hunk 3]
```diff
@@ -8,25 +8,68 @@ export default class FaqMerchantsDirectDebitEn extends React.Component {
...
-        </h3>
-        <p className='para'>
-          <a href='/direct-debit/guarantee/'
-          className='u-link-color-p u-text-underline'>The Direct Debit Guarantee</a> offers protection to customers paying by
-          Direct Debit in the rare event that there is an error in a payment.
-        </p>
-        <p className='para'>
-          If a customer receives a refund they are not entitled to, they must pay it back at the organisation's
-          request. Dispute resolution takes place outside of the Direct Debit scheme.
-        </p>
```
Comment (harrison) [inline app/pages/faq/merchants/direct-debit/faq-merchants-direct-debit.en.js]: same as the other faq, only a header left for `en-IE`

File: app/pages/faq/merchants/international-payments/faq-merchants-international-payments.en.js [hunk 4]
```diff
@@ -9,24 +11,34 @@ export default class FaqMerchantsInternationalPayments extends React.Component {
...
-        <h3 className='section-heading u-text-heading-light u-color-heading u-margin-Vm u-text-s'>
-          When can I collect from Europe?
-        </h3>
-        <p className='para'>
-          With GoCardless, any merchant registered in the U.K. or Ireland can collect from the Single Europe Payments
-          Area. You will need to have a Euro account open in the UK or a Eurozone country.
-        </p>
-        <p className='para'>
-          If you take more than 500 payments a month and want to use our Pro solution, you can be registered anywhere in the world.
-        </p>
```
Comment (harrison) [inline app/pages/faq/merchants/international-payments/faq-merchants-international-payments.en.js]: same

File: app/pages/faq/merchants/customer-experience/faq-merchants-customer-experience.en.js (hunk 1 above)
Comment (harrison) [inline app/pages/faq/merchants/customer-experience/faq-merchants-customer-experience.en.js]: oh see what's going on here

File: app/pages/faq/merchants/direct-debit/faq-merchants-direct-debit.en.js (hunk 3 above)
Comment (harrison) [inline app/pages/faq/merchants/direct-debit/faq-merchants-direct-debit.en.js]: ignore

File: app/pages/faq/merchants/international-payments/faq-merchants-international-payments.en.js (hunk 4 above)
Comment (harrison) [inline app/pages/faq/merchants/international-payments/faq-merchants-international-payments.en.js]: ignore

File: app/pages/faq/merchants/partners/faq-merchants-partners.en.js [hunk 5]
```diff
@@ -8,7 +8,7 @@ export default class FaqMerchantsPartnersEn extends React.Component {
 
   render() {
     return (
-      <Translation locales='en'>
+      <Translation locales='en-GB'>
```
Comment (harrison) [inline app/pages/faq/merchants/partners/faq-merchants-partners.en.js]: set on route?

**Comment by Ken-Lo** [comment]
@harrison Thanks! Have just tidied up following your comments & fixed the lint errors. Not sure why it's now failing test with `Error: Module did not self-register.`? Any other fixes or does the rest of the code look OK?

@octave-auger @adeclosset Would be great if you guys could double-check the content before we merge!

Thanks all!

**Comment by harrison** [comment]
Might be a random failure, retrying

**Comment by hirokitakeuchi** [comment]
Hey all - keen to get this deployed! What is left to do before we can?

**Comment by harrison** [comment]
@Ken-Lo rebase and merge?

**Comment by Ken-Lo** [comment]
@hirokitakeuchi @harrison Sounds good, will rebase and merge in the next few mins (assuming there are no other comments or changes for the code).

**Comment by octave-auger** [comment]
To give you an update on my tasks to clean this up:
- One FAQ section is still based on UK; I had a technical issue (maybe
nesting translations too much) so I need to double check that and push
- Business process: I'm emailing Tim to make sure we have the right phone
numbers and Salesforce flow (for now they'll all be dealt with by the UK
team)

**Pull request merged**